src/config.ts

```diff
@@ -100,3 +109,27 @@ async function loadAwsCredentials(profile: string = 'default'): Promise<AWSConfi
     sessionToken: sessionToken,
   };
 }
+
+async function loadSsoAwsCredentials(profile: string = 'default', region: string = 'eu-west-1'): Promise<AWSConfig['credentials']> {
+    const awsCredentials = fromSSO({
+      profile: profile,
+    });
+
+    const ssoCredentials = new SSOClient({
+      region: region,
+      credentials: awsCredentials,   
+    });
+
+    const credentials = await ssoCredentials.config.credentials();
```

Comment:
There are TypeScript errors in this section. Type definitions do not seem to match. Can you please have a look?

<img width="881" alt="image" src="https://github.com/kamranahmedse/aws-cost-cli/assets/4921183/d0f3bed8-6898-486e-a2c3-3010d29bd155">
cost-cli/assets/4921183/da686d9a-eee3-4a4a-b670-0d6e5f52bf0d">